CHANGELOG.md

```diff
@@ -194,31 +253,40 @@ the code was deployed.
 - Increase life of login cookie.
 
 ### Fixed
+
 - Concurrency issue.
 
 ### Removed
+
 - Global state.
 - Old Dashboard.
 
 ## [1.0-chatbot] - 2018-12-19
+
 ### Added
+
 - Initial chatbot.
 
 ## [1.2-pi-heartbeat] - 2019-07-16
+
 ### Added
+
 - Travis CI to run automated tests.
 - Heartbeat code to regularly ping the Flic Hub and send resulting data to the server.
 - Wifi connectivity for the Raspberry Pi.
 
 ### Changed
+
 - Specify separate test requirements for pip.
 - pi install script uses config file instead of interactive input.
 
 ## [1.0-pi-heartbeat] - 2019-01-11
+
 ### Added
+
 - Initial pi.
 
-[Unreleased]: https://github.com/bravetechnologycoop/BraveButtons/compare/v3.3.0...HEAD
+[unreleased]: https://github.com/bravetechnologycoop/BraveButtons/compare/v3.3.0...HEAD
```

Comment:
Note: This case change was done by prettier. The link still seems to work even with the different case.